tools/cldr-code/src/main/java/org/unicode/cldr/util/PathHeader.java

```diff
@@ -91,10 +91,10 @@ public enum SurveyToolStatus {
          * Changes are allowed as READ_WRITE, but field is always displayed as LTR, even in RTL
          * locales (used for patterns).
          */
-        LTR_ALWAYS;
+        LTR_ALWAYS
```

Comment:
changes in this file are all refactoring to fix compiler warnings, except for the addition of Travel_Places2 and  Objects2 and a TODO comment for adding Volume2